### PR TITLE
AC-620: Make UI more consistent

### PR DESCRIPTION
**Pull request by deepak140596**
## Description of what I changed

- Changed activity_patient_dashboard to CoordinatorLayout with a Nested Scroll view.
- Changed Patient Details Fragment layout to be more consistent and follow Material Guidelines
- Used @dimens as much as possible
- Added some margin dimens and text view sizes for a consistent layout
- Changed naming style for rows, now for every row layout, the name starts with row_*

## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-620

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [ ] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/activevisits/ActiveVisitsRecyclerViewAdapter.java [hunk 1]
```diff
@@ -48,7 +48,7 @@ public ActiveVisitsRecyclerViewAdapter(Context context, List<Visit> items) {
     @NonNull
```
Comment (deepak140596) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/activevisits/ActiveVisitsRecyclerViewAdapter.java]: Refactored name and Formatted using AS

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/addeditpatient/SimilarPatientsRecyclerViewAdapter.java [hunk 2]
```diff
@@ -39,9 +39,10 @@
```
Comment (deepak140596) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/addeditpatient/SimilarPatientsRecyclerViewAdapter.java]: Refactored name and formatted using AS

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/formentrypatientlist/FormEntryPatientListAdapter.java [hunk 3]
```diff
@@ -32,6 +32,7 @@
```
Comment (deepak140596) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/formentrypatientlist/FormEntryPatientListAdapter.java]: Refactored name and formatted using AS

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/lastviewedpatients/LastViewedPatientRecyclerViewAdapter.java [hunk 4]
```diff
@@ -43,6 +43,7 @@
```
Comment (deepak140596) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/lastviewedpatients/LastViewedPatientRecyclerViewAdapter.java]: Refactored name and formatted using AS

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/matchingpatients/MergePatientsRecycleViewAdapter.java [hunk 5]
```diff
@@ -38,7 +38,7 @@
 import androidx.core.content.ContextCompat;
```
Comment (deepak140596) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/matchingpatients/MergePatientsRecycleViewAdapter.java]: Refactored name and formatted using AS

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/patientdashboard/PatientDashboardActivity.java [hunk 6]
```diff
@@ -102,19 +107,21 @@ private void attachPresenterToFragment(Fragment fragment) {
...
             mPresenter = new PatientDashboardVitalsPresenter(id, ((PatientVitalsFragment) fragment));
+        } else if (fragment instanceof PatientChartsFragment) {
+            mPresenter = new PatientDashboardChartsPresenter(id, ((PatientChartsFragment) fragment));
         }
-        else if (fragment instanceof PatientChartsFragment){
-            mPresenter = new PatientDashboardChartsPresenter(id,((PatientChartsFragment) fragment));
-        }
+    }
+
+    public void setBackdropImage(Bitmap backdropImage, String patientName) {
```
Comment (deepak140596) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/patientdashboard/PatientDashboardActivity.java]: Setting backdrop image and its click action

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/patientdashboard/visits/PatientVisitsRecyclerViewAdapter.java [hunk 7]
```diff
@@ -44,7 +44,7 @@ public PatientVisitsRecyclerViewAdapter(PatientVisitsFragment context, List<Visi
     @NonNull
```
Comment (deepak140596) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/patientdashboard/visits/PatientVisitsRecyclerViewAdapter.java]: Refactored name and formatted using AS

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/syncedpatients/SyncedPatientsRecyclerViewAdapter.java [hunk 8]
```diff
@@ -76,7 +76,7 @@ public SyncedPatientsRecyclerViewAdapter(SyncedPatientsFragment context, List<Pa
     @NonNull
     @Override
     public SyncedPatientsRecyclerViewAdapter.PatientViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
-        View itemView = LayoutInflater.from(parent.getContext()).inflate(R.layout.find_synced_patients_row, parent, false);
```
Comment (deepak140596) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/syncedpatients/SyncedPatientsRecyclerViewAdapter.java]: Refactored name and formatted using AS

File: openmrs-client/src/main/java/org/openmrs/mobile/application/OpenMRSInflater.java [hunk 9]
```diff
@@ -32,7 +32,7 @@ public OpenMRSInflater(LayoutInflater inflater) {
     }
```
Comment (deepak140596) [inline openmrs-client/src/main/java/org/openmrs/mobile/application/OpenMRSInflater.java]: Refactored name and formatted using AS

**Comment by f4ww4z** [comment]
Looks good @deepak140596 ! Tested locally.

**Pull request merged**